docs/DeveloperGuide.md

```diff
@@ -370,8 +370,57 @@ The structure for executing an `addprop` command follows the flow as mentioned i
 #### Design considerations:
 No duplicate properties can be added to the property list. This means that no two properties with the same address can exist. We used name and price to identify a property in previous iterations, but later decided against it since in real life there could be identical properties with the exact same name and price. The only thing unique to the property would be the unit number recorded in the address.
 The entry time is added towards later of the development to help facilitate a more flexible implementation of the `sortprops` command.
+
+### Editing of buyers and properties
+
+#### Motivation
+The user may want to edit the details of a buyer or property after adding it to the application. For example, the user may want to change the budget range of a buyer after adding it to Cobb. 
+Or, the user may want to change the price of a property after adding it to Cobb.
+
+#### Implementation
+The `EditBuyerCommand` and `EditPropertyCommand` class extends the `Command` class. They are used to edit the details of a buyer or property, respectively.
+Both commands allow the user to change any of the fields of a buyer or property. The commands expect at least one flag to be edited, otherwise an error message will be displayed.
+When the edit command is inputted, the `EditBuyerCommandParser` and `EditPropertyCommandParser` classes are used to parse the user input and create the respective `EditBuyerCommand` and `EditPropertyCommand` objects.
+When these created command objects are executed by the `LogicManager`, the `EditBuyerCommand#execute(Model model)` or `EditPropertyCommand#execute(Model model)` methods are called. These methods will edit the buyer or property in the model, and return a `CommandResult` object.
+
+
+<div markdown="span" class="alert alert-primary">:exclamation: **Note:**
+To be more concise, we will be referring to both buyers and properties as entities in this section from here onwards.
```

Comment:
I think for some parts in the guide, we refer to buyers and properties as `entries` instead of `entities`, should I change all related parts to reflect `entities` instead?

Reply:
I think if it makes sense in your context sure! i did specify that its only for this section, but yea entities work better in my context